polkadot-protocol/architecture/index.md

```diff
@@ -14,11 +14,11 @@ Explore Polkadot's architecture, including the relay chain, parachains, and syst
 
 The following provides a brief overview of the role of each chain:
 
-- **Polkadot chain** - the central hub and main chain responsible for the overall security, consensus, and interoperability between all connected chains
+- [**Polkadot chain**](/polkadot-protocol/architecture/polkadot-chain/) - the central hub and main chain responsible for the overall security, consensus, and interoperability between all connected chains
```

Comment:
Just a bit unsure about this one, idk if it's redundant or not here  but I'd like to hear a second opinion

Reply:
I think it's fine cause it provides a high-level overview about what each one's role is so users can decide where to go from there